workflows/fastquorum.nf

```diff
@@ -92,10 +93,17 @@ workflow FASTQUORUM {
     MERGE_BAM(bam_to_merge.multiple, [[], []], [[], []])
     ch_versions = ch_versions.mix(MERGE_BAM.out.versions.first())
 
+    //
+    // MODULE: Run fgbio SortBam to re-sort into TemplateCoordinate.  This can be removed when samtools is released
+    // with the following bugfix: https://github.com/samtools/samtools/pull/2062
+    //
+    SORTBAM(MERGE_BAM.out.bam)
```

Comment:
This is unfortunate because it will probably create a performance/speed regression but I do see we can move off it once a new samtools release is made!

Reply:
Yes, it isn't ideal but better than a non-functional pipeline.